gaia-header.js

```diff
@@ -761,9 +755,11 @@ module.exports = component.register('gaia-header', {
   }
 
   [action=close] .action-button:before { content: 'close' }
-  [action=back] .action-button:before { content: 'back' }
   [action=menu] .action-button:before { content: 'menu' }
 
+  [action=back]:-moz-dir(ltr) .action-button:before { content: 'back' }
+  [action=back]:-moz-dir(rtl) .action-button:before { content: 'forward' }
```

Comment:
Correct. I built a workaround into gaia-component.js though. You can use [`<gaia-switch>`](https://github.com/gaia-components/gaia-switch/blob/master/gaia-switch.js) as an example of how to use it.

You must add `rtl: true` property to the prototype and then listen for the `'dirchanged'` event on the document. I'm not sure if this has been used in production yet, so I'd be interested to get your feedback.

If for some reason the `'dirchanged'` event doesn't trigger in Gaia, we may have to listen to something more `l10n.js` specific like `'localized'` event.